### ISSUE TITLE
NVDA Doesn't announce newly added number and bullet in MS word, as well as different sections:

### ISSUE THREAD

**Issue by kb1409** [nvaccess/nvda#15855]
In the recent updates of NVDA, there are some problems in using MS word.
1. When we create a list in MS word, i.e. the bullets or numbers, and when we give the "Enter" key, the NVDA doesn't read out the newly added number in the list. So, it confuses us whether the new number is added or not.
2. While creating different sections in MS word, the NVDA is not reading it as a new section, and also the section number is not being read.
